services/src/test/java/org/keycloak/protocol/docker/mapper/TestAuthenticatedClientSessionModel.java

```diff
@@ -0,0 +1,119 @@
+package org.keycloak.protocol.docker.mapper;
```

Comment:
Is it possible to move this class to `org.keycloak.protocol` package (and likely make it public), so it can be eventually used by other "unit tests" for protocol mappers? It doesn't have anything docker-protocol specific, so can be more shared.